samples/Add Voice Calling/Web/JavaScript/src/MakeCall/Login.js

```diff
@@ -0,0 +1,175 @@
+// © Microsoft Corporation. All rights reserved.
+
+import React from "react";
+import { PrimaryButton } from 'office-ui-fabric-react'
+import { utils } from "../Utils/Utils";
+export default class Login extends React.Component {
+    constructor(props) {
+        super(props);
+        this.userDetails = null;
+
+        this.state = {
+            showUserProvisioningAndSdkInitializationCode: false,
+            showSpinner: false,
+            disableInitializeButton: false,
+            loggedIn: false
+        }
+    }
+
+    provisionNewUser = async () => {
+        try {
+            this.setState({ showSpinner: true});
+            this.userDetails = await utils.provisionNewUser();
+            this.setState({ id: utils.getIdentifierText(this.userDetails.user) });
+            await this.props.onLoggedIn({ id: this.state.id, token: this.userDetails.token });
+            this.setState({ loggedIn: true });
+        } catch (Error) {
+            console.log(Error);
+        } finally {
+            this.setState({ disableInitializeButton: true, showSpinner: false });
+        }
+    }
+
+    render() {
+        const userProvisioningAndSdkInitializationCode = `
+/**************************************************************************************
+ *   User token provisioning service should be set up in a trusted backend service.   *
+ *   Client applications will make requests to this service for gettings tokens.      *
+ **************************************************************************************/
+import  { CommunicationIdentityClient } from @azure/communication-administration;
+const communicationIdentityClient = new CommunicationIdentityClient(<RESOURCE CONNECTION STRING>);
+app.get('/tokens/provisionUser', async (request, response) => {
+    try {
+        const { communicationUserId } = await communicationIdentityClient.createUser();
+        const tokenResponse = await communicationIdentityClient.issueToken({ communicationUserId }, ['voip']);
+        response.json(tokenResponse);
+    } catch(error) {
+        console.log(error);
+    }
+});
+
+/********************************************************************************************************
+ *   Client application initializing the ACS Calling Client Web SDK after receiving token from service   *
+ *********************************************************************************************************/
+import { 
```

Comment:
these imports make sense in TS , not in JS

Reply:
same below, e.g. CallClientOptions is a TS type, all these things will be `undefined` when imported in JS